jena/src/main/scala/org/w3/banana/jena/io/JenaRDFReader.scala

```diff
@@ -62,10 +62,22 @@ object JenaRDFReader {
     }
   }
 
+  def makeAnyRDFReader = new AnyRDFReader[Jena, Try] {
+    /**
+     * Read triples from the given location. The syntax is determined from input source URI
+     *  (content negotiation or extension).
+     */
+    def readAnyRDFSyntax(uri: String): Try[Jena#Graph] = Try {
+      val model = RDFDataMgr.loadModel(uri)
```

Comment:
One problem lies here. Using the built in RDF loaders from Jena or sesame is like going back to the stone ages. All IO should be in a separate package, and not tied into the RDF core, as that can be done so much better with Scala.